sls-gallery.yml

```diff
@@ -0,0 +1,118 @@
+        omero.jvmcfg.percent.indexer: 20
+        omero.jvmcfg.percent.pixeldata: 30
+        omero.ldap.config: True
+        omero.ldap.base: "{{ omero_server_ldap_base }}"
+        omero.ldap.username: cn=ldapuser,ou=system,o=dundee
+        omero.ldap.user_filter: "(objectClass=person)"
+        omero.ldap.group_filter: "(&(objectClass=groupOfUniqueNames)(cn=omedev))"
+        omero.ldap.group_mapping: name=cn
+        omero.ldap.new_user_group: :query:(uniqueMember=@{dn})
+        omero.ldap.urls: "{{ omero_server_ldap_urls }}"
+        omero.mail.config: true
+        omero.mail.from: "qa@openmicroscopy.org.uk"
+        omero.mail.host: "smtp.dundee.ac.uk"
+        omero.pixeldata.max_plane_height: 5120
+        omero.pixeldata.max_plane_width: 5120
+        omero.security.keyStore: "/etc/pki/java/cacerts"
+        omero.security.trustStore: "/etc/pki/java/cacerts"
+        omero.security.keyStorePassword: "changeit"
+        omero.security.trustStorePassword: "changeit"
+
+    - role: ome.ldap-cert
+      ldap_host: "ldap.lifesci.dundee.ac.uk"
+
+    - role: ome.redis
+
+    - role: ome.nginx
+
+    - role: ome.nginx-configure
+      omero_web_conf: "omero/files/sls-gallery-omero-web.conf"
+
+    - role: ome.omero-web
+      ice_python_wheel: https://github.com/ome/zeroc-ice-py-centos7/releases/download/0.1.0/zeroc_ice-3.6.4-cp27-cp27mu-linux_x86_64.whl
+      omero_web_setup_nginx: False
+      omero_web_systemd_start: False
+      omero_web_release: latest
+      omero_web_config_set:
+        omero.web.server_list:
+          - ["{{ ansible_default_ipv4.address }}", 4064, "SLS Gallery"]
+        omero.web.prefix: '/ome-sls'
+        omero.web.static_url: '/ome-sls/static/'
+        omero.web.login_redirect:
+          redirect:
+            - webindex
+          viewname: "load_template"
+          query_string: "experimenter=-1"
+          args:
+            - userdata
+        omero.web.ui.top_links:
+          - ["Image Gallery", "webindex", {"title": "Image Gallery"}]
+          - ["HELP", "https://help.openmicroscopy.org/web-client.html", {"title": "Help", "target": "new"}]
+          - ["SLS Homepage", "https://www.lifesci.dundee.ac.uk/", {"title": "SLS Homepage", "target": "new"}]
+        omero.web.caches:
+          default:
+            BACKEND: django_redis.cache.RedisCache
+            LOCATION: redis://127.0.0.1:6379/0
+        omero.web.session_engine: django.contrib.sessions.backends.cache
+        omero.web.apps:
+          - "omero_iviewer"
+        omero.web.open_with:
+          - ["Image viewer", "webgateway", {"supported_objects": ["image"], "script_url": "webclient/javascript/ome.openwith_viewer.js"}]
+          - ["omero_iviewer", "omero_iviewer_index", {"supported_objects": ["images", "dataset", "well"], "script_url": "omero_iviewer/openwith.js", "label": "OMERO.iviewer"}]
+        omero.web.viewer.view: omero_iviewer.views.index
+
+    - role: ome.omero-web-apps
+      omero_web_app_names:
```

Comment:
typo: should be `omero_web_apps_names`. Though this will then conflict with the earlier `omero.web.apps`

Reply:
This is still https://github.com/mtbc/ansible-role-omero-web-apps - yet to investigate https://github.com/ome/prod-playbooks/pull/182#issuecomment-554446397.